src/e2e-test/features/BQExecuter/BigQuery Executer_WithMacro.feature

```diff
@@ -0,0 +1,29 @@
+@BQExecute121
+Feature: BigQuery Executes - Executes a SQL query in BigQuery Execute with macro arguments
```

Comment:
-> BigQueryExecute - Verify successful data transfer to BQ table using BQExecute plugin with macro arguments